### DISCUSSION TITLE
Orders GET - Returning partial data

### DISCUSSION THREAD

**Discussion by santichuit** [Sellbrite/Sellbrite-API#19] [Q&A]
Hi All,
I tried to get all the others within 2 dates and it retrieves only half of them.
Digging a bit more, I can see that for entire months, the result is empty.

https://api.sellbrite.com/v1/orders?page=1&min_ordered_at=2023-4-01&max_ordered_at=2023-4-30
While downloading all the orders from the UI, it shows 117 orders.

Any advise please?

**Comment by hlee-godaddy** [comment]
Hi @santichuit,

Thank you for reaching out!

The date uses ISO 8601 which follows the YYYY-MM-DD format. Please try again using the following URL:

https://api.sellbrite.com/v1/orders?page=1&min_ordered_at=2023-04-01&max_ordered_at=2023-04-30

Let me know if this helps.

**Comment by santichuit** [comment]
Hi @hlee-godaddy 
Thanks for your prompt support. I'm still getting an empty set with that URL too.
